announcements/2021/week09.md

```diff
@@ -0,0 +1,29 @@
+# :zap: This week in provider-independent announcements (March 5, 2021) :zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed apps
+
+[NGINX IC v1.15.0](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v1.15.0/) upgrades the container images (controller container to v0.44.0 and kube-webhook-certgen to v1.5.1). It also removes conflicting admission webhook api versions...
```

Comment:
@QuentinBisson i think you removed conflicting admission  webhook, etc. for a specific reason? Maybe add one line here on the value of that? cc @giantswarm/app-squad-nginx

Reply:
https://github.com/giantswarm/nginx-ingress-controller-app/pull/178#issuecomment-784179339 explains it a bit more

```
Sure, so basically, the default validation webhook matching mode is Equivalent by default since webhook v1. This policy will ensure that the tree of apiversions will be valid according the the webhook.
```

Reply:
Nice one